client/src/main/java/com/influxdb/client/domain/InfluxQLQuery.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * The MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.influxdb.client.domain;
+
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nonnull;
+
+public class InfluxQLQuery {
+    private final String command;
+    private final String database;
+    private String retentionPolicy;
+    private Precision precision;
+
+    public InfluxQLQuery(@Nonnull final String command, @Nonnull final String database) {
+        this.command = command;
+        this.database = database;
+    }
+
+    public String getCommand() {
+        return command;
+    }
+
+    public String getDatabase() {
+        return database;
+    }
+
+    public String getRetentionPolicy() {
+        return retentionPolicy;
+    }
+
+    public InfluxQLQuery setRetentionPolicy(final String retentionPolicy) {
+        this.retentionPolicy = retentionPolicy;
+        return this;
+    }
+
+    public Precision getPrecision() {
+        return precision;
+    }
+
+    public InfluxQLQuery setPrecision(final Precision precision) {
```

Comment:
Please add `@Nullable` annotation:

```suggestion
    public InfluxQLQuery setPrecision(@Nullable final Precision precision) {
```